docs/pipeline_configuration.md

```diff
@@ -0,0 +1,75 @@
+## Overview
+Creating pipelines in Conduit is a simple process with many steps to define the resources and their configurations, this
+could be done through API or Conduit's UI. But to make this process even simpler and to make it possible to load 
+pre-defined pipelines, we introduce **_Pipeline Configurations_**.
+
+Using **_Pipeline Configurations_** you will define all the pipelines you want to be provisioned into Conduit at startup,
+only by creating `.yml` files that define all the pipelines, with their connectors, processors, and configurations.
+
+## Getting started
+create a folder called _"pipelines"_ at the same level as your Conduit binary file, add all your `.yml` files there, 
+then run Conduit using the cmd command: 
+```
+./conduit
+```
+If you have your YAML files at a different directory, simply run conduit with the cli flag `pipelines.path` and point to 
+your directory, ex: 
+```
+./conduit -pipeline.path ../my-directory
+```
+
+_**Note**_: provisioned pipelines are **immutable**, any updates needed on a provisioned pipeline have to be done through
+the configuration file it was provisioned from. you can only control stopping and starting a pipeline through the UI and API.
+
+### YAML file format
+The file in general has two root keys, the "version", and the "pipelines" map. The map consists of other elements like 
```

Comment:
I suggest using a backtick when quoting fields (e.g. `version` instead of "version").